content/blog/2020-04-27-moving-to-the-matrix/index.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Goodbye Slack: GNU Radio is moving into the Matrix"
+author: "Martin Braun"
+date: "2020-04-27"
+sponsored: "0"
+aliases: ["blog/moving-to-the-matrix"]
+---
+
+With the growth of a project there come changes. For GNU Radio, a big change is
+in the pipeline too: We're moving from our chat service from Slack to
+[Matrix](https://matrix.gnuradio.org).
+Untypically for us, this decision came about pretty quickly, but without much
+community involvement. Let me go through why did this, how we reached a decision,
+and, most importantly, how you can join the new chat service as well.
+
+<!--more-->
+
+
+## Stop talking, and let me chat: Instructions for the impatient
+
+If you're already done reading, all you need to do is head over to
+[matrix.gnuradio.org](https://matrix.gnuradio.org) and create an account. Once
+you're registered, you will find a chat service that looks similar to our Slack
+domain. Some of the existing channels on Slack have already been re-established
+on Matrix, and some groups, like the documentation team lead by Marc Lichtman,
+have already moved their base of operations to Matrix.
+
+Other channels are still moving, and we expect the move to take a little bit.
+But technically speaking, there's nothing from stopping you to join right now!
+
+Once you've registered, you can also join our chat server from your phone using
```

Comment:
We need to add the info that the homeserver url is `gnuradio.matrix.ungleich.cloud`. Would it make more sense to direct people to https://wiki.gnuradio.org/index.php/Chat and document it there?

Reply:
I would prefer doing it on the wiki; that's easier to maintain. I'll add the wiki link.